components/dashboard/components/CourseConnections/CourseConnectionsView.jsx

```diff
@@ -462,6 +471,126 @@ class CourseConnectionsView extends React.Component {
         };
     }
 
+    /**
+     * Prepares the aria description for a user node
+     *
+     * @param {InteractionContext} data - containing interaction counts for the node
+     *
+     * @returns {string} containing the aria label for the node
+     */
+    getUserAriaLabel(data) {
+        const ariaDescription = ['You\'ve'];
+
+        if (data.contextType === 'you') {
+            ariaDescription.push('made');
+        }
+        else {
+            ariaDescription.push('had');
+        }
+
+        // Add aggregate interaction count to aria description
+        ariaDescription.push(getInteractionCountPhrase('aggregate', data.getInteractionAggregate()));
+
+        // user in course node
+        if (data.contextType === 'course') {
+            ariaDescription.push(`in the course with ${data.username}, including:`);
+        }
+
+        // user in plg team node
+        else if (data.contextType === 'plg') {
+            ariaDescription.push(`in your peer learning group with ${data.username}, including:`);
+        }
+
+        // user in capstone team node
+        else if (data.contextType === 'capstone') {
+            ariaDescription.push(`in your capstone team with ${data.username}, including:`);
+        }
+
+        // 'you' node
+        else if (data.contextType === 'you') {
+            ariaDescription.push('for the entire course, including:');
+        }
+
+        const counts = [];
+
+        // Add reaction count
+        counts.push(getInteractionCountPhrase('reaction', data.getInteractionCount('Reaction')));
```

Comment:
Wouldn't it make sense to have the 1st parameter to the `getInteractionCountPhrase` function actually be an InteractionType, rather than a "random" string. In particular you could then do this:
```
const countsToInclude = [
    'Reaction',
    'Reply',
    'Mention',
    'Post',
    'DirectMessage',
];

const counts = countsToInclude.map((interactionType) => getInteractionCountPhrase(
    interactionType,
    data.getTypeAggregateTotals().getCount(interactionType)
);
```